### PR TITLE
Add difference view to earnings charts

### LINKED ISSUE #526: Add a radio button to change household impact charts to delta
Instead of baseline/reform values, the radio button will change the net income and MTR chart to the difference, in £ and pp respectively.

### PR DESCRIPTION
**Pull request by tolaouk**
fixes #526

### CODE REVIEW

**Comment by tolaouk** [comment]
# case with reform:
image

image
image

# case with no reform: (no radio button)
image

File: policyengine-client/src/policyengine/pages/household/earningsCharts.jsx [hunk 1]
```diff
@@ -55,23 +57,46 @@ export default class AccountingTable extends React.Component {
...
-        );
+        const reformExists = Object.keys(this.context.getPolicyJSONPayload()).length > 0;
+        if (reformExists) {
+            return (
+                <>
+                    <Spacing />
+                    <Row>
+                        <div className="justify-content-center d-flex">
+                            <Radio.Group defaultValue={true} buttonStyle="solid" onChange={() => this.setState({ show_difference: !this.state.show_difference })} >
+                                <Radio.Button value={true}>Baseline and Reform Chart</Radio.Button>
```
Comment (nikhilwoodruff) [inline policyengine-client/src/policyengine/pages/household/earningsCharts.jsx:74]:
```suggestion
                                <Radio.Button value={true}>Baseline and reform</Radio.Button>
```

File: policyengine-client/src/policyengine/pages/household/earningsCharts.jsx [hunk 2]
```diff
@@ -55,23 +57,46 @@ export default class AccountingTable extends React.Component {
...
+                                <Radio.Button value={false}>Difference Chart</Radio.Button>
```
Comment (nikhilwoodruff) [inline policyengine-client/src/policyengine/pages/household/earningsCharts.jsx:75]:
```suggestion
                                <Radio.Button value={false}>Difference</Radio.Button>
```

**Review by nikhilwoodruff — COMMENTED** [review]
Looks great @tolaouk - I think just the name change, and could you also center the radio button like the others? Thanks!

**Pull request merged**